types/office-js-preview/index.d.ts

```diff
@@ -125269,35 +125269,113 @@ declare namespace PowerPoint {
          */
         unsupported = "Unsupported",
         /**
-         * The shape is an image
+         * The shape is an image.
          * @remarks
          * [Api set: PowerPointApi 1.4]
          */
         image = "Image",
         /**
-         * The shape is a geometric shape such as rectangle
+         * The shape is a geometric shape such as rectangle.
          * @remarks
          * [Api set: PowerPointApi 1.4]
          */
         geometricShape = "GeometricShape",
         /**
-         * The shape is a group shape which contains sub-shapes
+         * The shape is a group shape which contains sub-shapes.
          * @remarks
          * [Api set: PowerPointApi 1.4]
          */
         group = "Group",
         /**
-         * The shape is a line
+         * The shape is a line.
          * @remarks
          * [Api set: PowerPointApi 1.4]
          */
         line = "Line",
         /**
-         * The shape is a table
+         * The shape is a table.
          * @remarks
          * [Api set: PowerPointApi 1.4]
          */
         table = "Table",
+        /**
+         * The shape is a callout.
+         * @remarks
+         * [Api set: PowerPointApi 1.4]
```

Comment:
How are we getting so many new 1.4 APIs?

Reply:
checking with the FC